cmd/create/cluster/cmd.go

```diff
@@ -2611,6 +2632,38 @@ func run(cmd *cobra.Command, _ []string) {
 				os.Exit(1)
 			}
 
+			if interactive.Enabled() && !isGPULimitsInputSet && len(gpuLimits) == 0 {
```

Comment:
if user provided an empty list in the CLI we can skip the interactive request
```suggestion
			if interactive.Enabled() && !isGPULimitsInputSet {
```